defaults/main.yml

```
@@ -5,12 +5,15 @@ ethv_ssh_port: "22"
 # Number of validators
 ethv_number_validators: "1"
 
-# The http endpoint for an Ethereum 1.0 node (e.g. infura)
-ethv_10_http_provider: "http://127.0.0.1:8545"
-
 # Geth cache
 ethv_10_cache: "1024"
 
+# The http endpoint for an Ethereum 1.0 node (e.g. infura), this is for prysm only (below is for lighthouse)
+ethv_10_http_provider: "http://127.0.0.1:8545"
+
+# The below is for lighthouse only (the above is for prysm) as it currently supports fallback.
+ethv_10_http_providers: "http://127.0.0.1:8545"
```

and then:

```suggestion
ethv_10_lighthouse_http_providers: "http://127.0.0.1:8545"
```

This also looks like it could be a list? I read somewhere that lighthouse supports multiple endpoints, so this could be also:

```yaml
ethv_10_http_providers:
  - "http://127.0.0.1:8545"
```

and then the proper handling when using the variable in the task/template

---

I've heard prysm are adding support for fallback web3 endpoints. Once this is done we will be able to use ethv_10_http_providers and remove ethv_10_http_provider entirely so I don't know if it's cleaner to leave them as is.

---

Lighthouse does indeed take more than one provider and that's how I currently use it. I'm passing a comma separated string for simplicity. I agree that the list format would be cleaner.